docs/symbols-onEmit.md

```diff
@@ -0,0 +1,32 @@
+@typedef {function(function(*))} can-symbol/symbols/onEmit can.onEmit
+@parent can-symbol/symbols/observe
+@description Defines how observables can listen to the object's value whenever it is emitted.
+
+@signature `@@can.onEmit( handler(newValue) )`
+
+The `@@@@can.onEmit` symbol points to a function that registers 
+ `handler` to be called back with the new value of the object when it
+ is emitted.  
+
+```
+var obj = function(value) {
+	if(arguments.length >= 1) {
+		obj.currentValue = value;
+		obj.handlers.forEach(function(handler){
+			handler.call(obj, value);
+		});
+	} else {
+		return obj.currentValue;
+	}
+};
+
+obj[canSymbol.for("can.onEmit")] = function(handler){
+	if(!obj.handlers) {
+		obj.handlers = [];
+	}
+	obj.handlers.push(handler);
+}
```

Comment:
One thing I’d like to see in these docs is _when_ a value is emitted. A text explanation of what causes an “emit” would suffice, or if this example could be extended to show something that causes the emit, that would be good too.